dry_adiabatic_adjust/dadadj_apply_qv_tendency.meta

```diff
@@ -0,0 +1,42 @@
+[ccpp-table-properties]
+  name = dadadj_apply_qv_tendency
+  type = scheme
+#########################################################
+[ccpp-arg-table]
+  name = dadadj_apply_qv_tendency_run
+  type = scheme
+[ q_tend ]
+  standard_name = tendency_of_water_vapor_mixing_ratio_wrt_moist_air_and_condensed_water
+  units = kg kg-1 s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ state_q ]
+  standard_name = water_vapor_mixing_ratio_wrt_moist_air_and_condensed_water
+  long_name = mass mixing ratio of water vapor / dry air
```

Comment:
I think this long name needs to be updated:

```suggestion
  long_name = mass mixing ratio of water vapor / moist air and condensed water
```

Reply:
Updated long_name